static/js/log-results-grid.js

```diff
@@ -61,6 +70,35 @@ class ReadOnlyCellEditor {
     destroy() {
         this.eInput.classList.remove(cellEditingClass);
     }
+    addCopyIcon() {
+        // Remove any existing copy icons
+        $('.copy-icon').remove();
+      
+        // Add copy icon to the textarea
+        $('.copyable').each(function() {
+          var copyIcon = $('<span style="margin-right: 14px;" class="copy-icon"></span>');
```

Comment:
Looks good!
Just remove the right-margin.

Reply:
The icon is neither visible nor operatable due to the scroll bar which appears on scrolling if we don't keep the margin.

Reply:
Oh, okay. The scroll bar was not visible to me.